### DISCUSSION TITLE
How to use the YouTube Shorts detection feature

### DISCUSSION THREAD

**Discussion by Fl0rent** [cn-tools/cntools_FreshRssExtensions#27] [Q&A]
I am trying to use the YouTube Shorts detection feature. So far, without success. 

I don't really understand what settings I can adjust? (I see options to archive or mark as read) but this doesn't seem to work. 
How can I add, for example, the duration of the Shorts (in seconds, minutes...)?

And, thank you for your extension; it is essential for my workflow. 👍

![Capture d’écran 2024-07-26 à 09 56 19](https://github.com/user-attachments/assets/fa6f9166-52d4-46ea-8a95-aa41d11be1a6)

**Comment by cn-tools** [comment]
A few days ago i added many maschine-translatied languages.
I hope your favorite language is available now and you can config the extension as you need.

Otherwise let me know and i will help you.
